backends/memory/memory_backend.go

```diff
@@ -96,22 +96,18 @@ func (m *MemBackend) Enqueue(ctx context.Context, job *jobs.Job) (jobID string,
 		job.RunAfter = now
 	}
 
-	if job.Queue == "" {
-		err = jobs.ErrNoQueueSpecified
-		return
-	}
-
 	err = jobs.FingerprintJob(job)
 	if err != nil {
 		return
 	}
 
 	// if the job fingerprint is already known, don't queue the job
-	if _, found := m.fingerprints.Load(job.Fingerprint); found {
+	if _, found := m.fingerprints.Load(job.Fingerprint); found && job.Override {
```

Comment:
Should this be `!job.Override`? i.e. If the job is already found, and the override flag is not set, return that the job is duplicate.